docs/expo.md

```diff
@@ -0,0 +1,45 @@
+## Guide for apps created using the Expo CLI
+
+### Overview
+Expo projects can use both third-party React Native libraries with native code and your own custom native code. Creating a development build allows you to include your specific native dependencies and customisations.
+
+- Initializing the SDKs should be done in native code inside your AppDelegate (iOS) and MainApplication (Android). 
+
+- By default, Expo projects use Continuous Native Generation (CNG). This means that projects do not have android and ios directories containing the native code and configuration. You can opt out of CNG and directly manage the code and configuration inside your android and ios directories.
+
+### Installation
+
+- To generate these directories, run `npx expo prebuild` or compile your app locally (`npx expo run android` or `npx expo run ios`).
+
+```bash
+# Build your native Android project
+npx expo run:android
+# Build your native iOS project
+npx expo run:ios
+```
+
+Follow the [Installation Guide](../README.md#Installation) to install Adobe SDKs once the the android and ios directories are generated.
```

Comment:
The `AppDelegate` and `MainApplication` code generated by Expo CLI differs slightly from the sample code provided (for the bare RN projects) in the [Initializing](https://github.com/adobe/aepsdk-react-native/blob/main/README.md#initializing) section. Not all React Native developers are well-versed in the native implementations of Android and iOS.So, could we also revise the "Initializing" section to showcase how to modify the Expo-generated `AppDelegate` and `MainApplication` code?

Reply:
@ishita-gambhir-adobe 
How the AppDelegate setup for Expo CLI?  Do we need to use the Bridge files or just some update in AppDelegate?  Please update it, and I will run through it as a new User prespective.

Reply:
Also, in this setup, any modifications need to make for Xcode as the bare react native apps?